packages/nextjs/app/debug/_components/contract/ReadOnlyFunctionForm.tsx

```diff
@@ -38,7 +38,7 @@ export const ReadOnlyFunctionForm = ({
     functionName: abiFunction.name,
     abi: [...abi],
     args: inputValue ? inputValue.flat(Infinity) : [],
-    enabled: false,
```

Comment:
is this set by default to `false` on starknet-react ? if not, the idea of having this is that we dont execute many `FAILED` calls on first render , for example would affect if we have longer contracts or RPCs with limited bandwidth. any way we can get around this ?

Reply:
Default is `true`. the thing is that this is a wrapper around  `react-query`'s `useQuery`. The supposed behaviour is that to lazy load, this flag is set to `false`. However, for some reason in here requires calling `refetch` twice. 

One alternative would be to use input value as enabled flag? that way it's true only when input value is available, this way we can skip the `FAILED` calls

discussion by react-query team: https://github.com/TanStack/query/discussions/1205#discussioncomment-109277

Reply:
yes that sounds like a good approach, avoiding the FAILED calls